rj_gameplay/rj_gameplay/action/move.py

```diff
@@ -49,7 +49,7 @@ def tick(self, intent: msg.RobotIntent) -> msg.RobotIntent:
             path_command.override_angle=[self.face_angle]
 
         if(self.face_point is not None):
-            path_command.override_face_point=[Point(self.face_point[0], self.face_point[1])]
+            path_command.override_face_point=[Point(x=self.face_point[0], y=self.face_point[1])]
```

Comment:
@HussainGynai this is a fix to the move action and should probably be pulled into gameplay_node_update

Reply:
Nit: we should have a conversion helper between numpy and Point/Pose/Twist types, as we will be doing that a lot.